package.json

```diff
@@ -21,5 +21,9 @@
   "bugs": {
     "url": "https://github.com/cipherchase/cipherchase/issues"
   },
-  "homepage": "https://github.com/cipherchase/cipherchase#readme"
+  "homepage": "https://github.com/cipherchase/cipherchase#readme",
+  "dependencies": {
```

Comment:
dotenv should be a Dev dependency, lets move it there.

Reply:
dotenv moved to devDependency